.github/workflows/build-and-test.yml

```diff
@@ -117,16 +144,33 @@ jobs:
            -DCMAKE_C_COMPILER_LAUNCHER=ccache                                                          \
            -DCMAKE_CXX_COMPILER_LAUNCHER=ccache                                                        \
            ${GITHUB_WORKSPACE}
-      - run: ccache -p
-      - run: ccache -z
-      - name: CMake Build
-        working-directory: ./build
+      - name: CMake Build (Ubuntu)
+        if: ${{ startsWith(matrix.config.os, 'ubuntu') }}
+        working-directory: ./build_${{ matrix.config.conan_profile }}
         run: cmake --build .
-      - name: CCache Stats
-        run: ccache -s
-      - name: CMake Test
-        working-directory: ./build
+      - name: CMake Test (Ubuntu)
+        if: ${{ startsWith(matrix.config.os, 'ubuntu') }}
+        working-directory: ./build_${{ matrix.config.conan_profile }}
         run: CTEST_OUTPUT_FAILURE=1 ARGS="-E IntegrationTest" cmake --build . --target test
+      - name: Install conan config (Windows)
+        if: ${{ startsWith(matrix.config.os, 'windows') }}
+        run: |
+          py -3 -m conans.conan config install .\third_party\conan\configs\windows\
+          py -3 -m conans.conan config install .\third_party\conan\configs\windows_ci\
+      - name: Build & Test (Windows)
+        if: ${{ startsWith(matrix.config.os, 'windows') }}
+        env:
+          Qt5_DIR: "C:\\Qt\\5.15.2\\msvc2019_64"
+          CMAKE_CXX_COMPILER_LAUNCHER: "ccache"
+          CMAKE_C_COMPILER_LAUNCHER: "ccache"
+        run: |
+          py -3 -m conans.conan install -pr:b ${{ matrix.config.conan_profile }} -pr:h ${{ matrix.config.conan_profile }} -if build/ --build outdated --update ./
```

Comment:
The build profile `-pr:b` should be hard coded to some release profile. It's the profile that we use to build things like cmake and protoc. `msvc2019_release` is probably the best fit here.

Reply:
(Of course this is kind of forward looking for when we have more than a single windows build with maybe different profiles.)